notebooks/01-Summary-statistics-and-plots.ipynb

```diff
@@ -0,0 +1,265 @@
+    "* [Matplotlib — Visualization with Python](https://matplotlib.org/)\n",
+    "* [Tutorials — Matplotlib](https://matplotlib.org/stable/tutorials/index)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e9925511-9e30-4c69-86cf-531cda8fea10",
+   "metadata": {},
+   "source": [
+    "## Load Libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b95d8fd7-44dd-42b2-af15-12302221e59a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "\n",
+    "import pandas as pd\n",
+    "\n",
+    "from topostats.plotting import TopoSum"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "46f8ee5e-428e-4b91-972a-45d478728f18",
+   "metadata": {},
+   "source": [
+    "## Load  `all_statistics.csv`\n",
+    "\n",
+    "You need to load your data to be able to work with it, this is best achieved by importing it using\n",
+    "[Pandas](https://pandas.pydata.org/). Here we use the `tests/resources/minicircle_default_all_statistics.csv` that is\n",
+    "part of the TopoStats repository and load it into the object called `df` (short for \"Data Frame\"). You will need to\n",
+    "change this path to reflect your output. \n",
+    "\n",
+    "Because `molecule_number` is unique to the `image` and `threshold` we set a multi-level index of these three"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "096b8ef4-08d9-4ed6-8009-a72328133720",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv(\"../tests/resources/minicircle_default_all_statistics.csv\")\n",
+    "df.set_index([\"image\", \"threshold\", \"molecule_number\"], inplace=True)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "451a88c5-46eb-4dbe-862d-b6aee2948376",
+   "metadata": {},
+   "source": [
+    "## Plotting with Pandas"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "91585e8a-8653-4328-b78c-9da32ade4fd2",
+   "metadata": {},
+   "source": [
+    "### Plotting Contour Lengths"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ab89b50a-ce17-4dc1-8139-5402b8298efb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df[\"contour_lengths\"].plot.hist(figsize=(16,9),\n",
+    "                                bins=20,\n",
+    "                                title=\"Contour Lengths\",\n",
+    "                                alpha=0.5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4239d57b-6840-4261-a98a-a1c25a8dec1d",
+   "metadata": {},
+   "source": [
+    "### Plotting End to End Distance of non-Circular grains"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cf4d92f0-b624-452c-9f51-52fe70dfde88",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df[df[\"circular\"] == False][\"end_to_end_distance\"].plot.hist(figsize=(16,9),\n",
```

Comment:
Perhaps add a comment here stating that we ignore entries that are marked as "circular" since they do not have end-to-end distances? Minor suggestion, feel free to ignore.